blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -47,7 +47,8 @@ Changes
 =======
 
 - Expose the total count and sum of durations metrics under the ``QueryStats``
```

Comment:
Maybe link to the doc section? As advocated here https://github.com/crate/crate/pull/8232/commits/ce5ffa09034f7039b3b834a657eb21ea1eb97340#diff-8918b36d12905ff85de79bcbdcf052aaR32